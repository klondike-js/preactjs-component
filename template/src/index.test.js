/* eslint-env node, jest */

import { h } from "preact";
import render from "preact-render-to-string";
import CustomComponent from "./index";


describe("CustomComponent Snapshot", () => {
  it("should render with content", () => {
    const tree = render(<CustomComponent />);
    expect(tree).toMatchSnapshot();
  });
});