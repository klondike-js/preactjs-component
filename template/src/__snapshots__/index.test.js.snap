// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomComponent Snapshot should render with content 1`] = `'<div class="custom-component">Custom Component</div>'`;